spec/system/admin_customize_themes_config_area_spec.rb

```diff
@@ -2,7 +2,9 @@
 
 describe "Admin Customize Themes Config Area Page", type: :system do
   fab!(:admin)
-  fab!(:theme) { Theme.where(component: false).first }
+  fab!(:theme)
+  fab!(:default_theme) { Theme.where(component: false, name: "Default").first }
+  fab!(:foundation_theme) { Theme.where(component: false, name: "Foundation").first }
```

Comment:
Can't we do something like this `Theme.find_by(id: Theme::CORE_THEMES[:foundation])`? Feels cleaner than hardcoding the name. I'd suggest that we extract the logic into a class method on `Theme`, i.e. `Theme.foundation_theme`.